llvm/include/llvm/CodeGen/MachineFunction.h

```diff
@@ -918,6 +918,13 @@ class LLVM_ABI MachineFunction {
   bool verify(Pass *p = nullptr, const char *Banner = nullptr,
               raw_ostream *OS = nullptr, bool AbortOnError = true) const;
 
+  /// For New Pass Manager: Run the current MachineFunction through the machine
+  /// code verifier, useful for debugger use.
+  /// \returns true if no problems were found.
+  bool verify(MachineFunctionAnalysisManager &MFAM,
+              const char *Banner = nullptr, raw_ostream *OS = nullptr,
+              bool AbortOnError = true) const;
```

Comment:
We probably should have one that separately takes all of the analyses (I thought I had a patch to do that for llvm-reduce)